spec/models/chargeback_rate_detail_currency_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe ChargebackRateDetailCurrency do
+  it "has a valid factory" do
+    expect(FactoryGirl.create(:chargeback_rate_detail_currency_EUR)).to be_valid
+  end
+  it "is invalid without a code" do
+    expect(FactoryGirl.build(:chargeback_rate_detail_currency_EUR, :code => nil)).not_to be_valid
```

Comment:
I generally don't test Rails validations unless it includes somewhat complex domain logic.  In this case, I would manually test that the min/max validations work while developing this feature and let Rails' test suite ensure that they don't break this functionality.